lib/pages/chat/chat_emoji_picker.dart

```diff
@@ -1,26 +1,41 @@
 import 'package:flutter/material.dart';
 import 'package:emoji_picker_flutter/emoji_picker_flutter.dart';
 import 'package:fluffychat/config/themes.dart';
-import 'chat.dart';
 
 class ChatEmojiPicker extends StatelessWidget {
-  final ChatController controller;
-  const ChatEmojiPicker(this.controller, {Key? key}) : super(key: key);
+  final ValueNotifier<bool> showEmojiPickerNotifier;
+  final void Function(Emoji? emoji) onEmojiSelected;
+  final void Function() emojiPickerBackspace;
+
+  const ChatEmojiPicker({
+    Key? key,
+    required this.showEmojiPickerNotifier,
+    required this.onEmojiSelected,
+    required this.emojiPickerBackspace,
+  }) : super(key: key);
 
   @override
   Widget build(BuildContext context) {
     return ValueListenableBuilder(
-      valueListenable: controller.showEmojiPickerNotifier,
+      valueListenable: showEmojiPickerNotifier,
       builder: (context, showEmojiPicker, _) {
         return AnimatedContainer(
+          decoration: const BoxDecoration(
+            border: Border(
+              top: BorderSide(
+                color: Colors.transparent,
+                width: 1,
+              ),
+            ),
+          ),
```

Comment:
Why we need it with color is transparent?

Reply:
no need the divider between the composer and emoji picker, if have, it looks ugly when run the animation